msteams-platform/toolkit/teams-toolkit-fundamentals.md

```diff
@@ -53,72 +53,72 @@ Let's take a tour of the topics covered in this document:
 
 ## Accounts
 
-To develop a Teams app, you need at least one Microsoft 365 account with a valid subscription. If you want to host your backend resources on Azure, an Azure account is also needed. Teams Toolkit support integrated experience to sign in, provision and deployment for Azure resources. You can [create a free Azure account](https://azure.microsoft.com/free/) before you start.
+To develop a Teams app you need a Microsoft 365 account with a valid subscription. If you want to host your backend resources on Azure, an Azure account is also needed. Teams Toolkit supports an integrated sign-in experience to provision and deploy Azure resources. If needed, you can [create a free Azure account](https://azure.microsoft.com/free/) before you start.
 
 ## Environment
 
-Teams Toolkit helps you to create and manage multiple environments, provision, and deploy artifacts to the target environment for Teams App.
+Teams Toolkit helps you to create and manage multiple environments, provision resources, and deploy artifacts to the target environment for the Teams app.
 
 ### TeamsFx Collaboration
 
-It allows developers and project owner to invite other collaborators to the TeamsFx project to debug, provision, and deploy the same TeamsFx project.
+It allows developers and project owners to invite other collaborators to the TeamsFx project to debug, provision, and deploy the project.
 
 ## Development
 
-Teams Toolkit helps you to create and customize your Teams app project that makes the Teams app development work simpler.
+Teams Toolkit helps you to create and customize your Teams app project, and makes simplifies Teams app development.
 
 ### Create a new Teams app
 
-It helps you to start with Teams app development by creating new Teams project using Teams Toolkit either by using **Create new project** or **Create from samples**.
+It helps you to get started with Teams app development by creating a new Teams project using Teams Toolkit either by using **Create new project** or **Create from samples**.
 
 ### Add capabilities
 
-It helps to add other required Teams capabilities to Teams app during development process.
+It helps to add other required Teams capabilities to Teams apps during the development process.
 
 ### Add cloud resources
 
-It helps you to optionally add cloud resources that fits your development needs.
+It helps you to optionally add cloud resources according to your development needs.
 
 ### Edit manifest file 
 
-It helps you to edit the Teams app integration with Teams client.
+It helps you to configure how the Teams app will be integrated with the Teams client.
```

Comment:
It helps you to configure the Teams apps that will be integrated with the Teams client.